src/Razor/Microsoft.NET.Sdk.Razor/src/ConcatenateCssFiles.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.AspNetCore.Razor.Tasks
+{
+    public class ConcatenateCssFiles : DotNetToolTask
+    {
+        [Required]
+        public ITaskItem[] FilesToProcess { get; set; }
+
+        [Required]
+        public string OutputFile { get; set; }
+
+        internal override string Command => "concatenatecss";
+
+        protected override string GenerateResponseFileCommands()
```

Comment:
You could do the concat in the task no?

Reply:
Circling back on this. This happens on the task directly.